space-cli/modules/addons/registry.go

```diff
@@ -57,15 +57,19 @@ func addRegistry(projectID string) error {
 			return utils.LogError("No projects found. Run this command after creating a project", err)
 		}
 
-		var projectIDOptions []string
-		for _, projectInfo := range projects {
-			projectIDOptions = append(projectIDOptions, projectInfo.ID)
+		projectID = projects[0].ID
+		if len(projects) > 1 {
```

Comment:
Check if length is zero? Throw an error requesting user to create a project first

Reply:
Already being done on line 56
Existing code